manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1

```

```suggestion
PackageVersion: 2.1.0
```
This must match Control Panel.